build/source/engine/run_oneGRU.f90

```diff
@@ -260,9 +271,7 @@ subroutine run_oneGRU(&
  call qOverland(&
                 ! input
                 model_decisions(iLookDECISIONS%subRouting)%iDecision,          &  ! intent(in): index for routing method
-                bvarData%var(iLookBVAR%basin__SurfaceRunoff)%dat(1),           &  ! intent(in): surface runoff (m s-1)
-                bvarData%var(iLookBVAR%basin__ColumnOutflow)%dat(1)/totalArea, &  ! intent(in): outflow from all "outlet" HRUs (those with no downstream HRU)
-                bvarData%var(iLookBVAR%basin__AquiferBaseflow)%dat(1),         &  ! intent(in): baseflow from the aquifer (m s-1)
```

Comment:
We do need these three arguments.

Reply:
can you elaborate?  these were removed to avoid calculating total runoff in 2 places ... the terms weren't used individually in qOverland.  I think the disconnect would be if there are more than one total runoff terms we want to calculate, eg on to use with the time delay, and one to use for something else.

Reply:
Basin runoff is not simply the sum of the HRU runoff in the case when there is a lateral flux across HRUs.

Reply:
We agreed to compute the total runoff in `run_oneGRU`. Hence these three arguments can be removed.